### DISCUSSION TITLE
Welcome to KernelSU Discussions!

### DISCUSSION THREAD

**Discussion by tiann** [tiann/KernelSU#947] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jeclkwu** [comment]
您好，我想问一下咱们能适配mipush吗？我试过，貌似不太支持，精简模块刷入也无效～
